Tests/RxSwift/pausableBufferedTests.swift

```diff
@@ -13,187 +13,175 @@ import RxSwiftExt
 import RxTest
 
 class PausableBufferedTests: XCTestCase {
-    
     let testError = NSError(domain: "dummyError", code: -232, userInfo: nil)
-    
     let scheduler = TestScheduler(initialClock: 0)
-    
+
     func testPausedNoSkip() {
-        
         let underlying = scheduler.createHotObservable([
             next(150, 1),
             next(210, 2),
             next(230, 3),
             next(301, 4),
             next(350, 5),
             next(399, 6),
-            completed(500),
+            completed(500)
```

Comment:
why do we drop trailing commas? I believe it's configurable from `SwiftLint`?

Reply:
Doesn't make sense have a trailing comma for an array ... It's like a sentence ending with a comma instead of a dot, don't you think ? 

Also, it's not across all tests which is even weirder. I personally highly prefer (and seems like its the standard) to have commas between elements, but not with a trailing one

Reply:
Its main purpose is cleaner diffs in the future. I personally prefer them (and I don't believe there is a way to say they are _not_ standard :D) but it's arguable of course ¯\_(ツ)_/¯

Reply:
I don't think it makes much difference in diffs 
And also - in the codebase itself (not tests), there are no trailing commas ... so how can we reason about trailing commas specifically in tests? What makes this special ?

Reply:
It does, no doubt :) so with a trailing comma it would look like;
```diff
   next(50, "alpha"),
+   completed(50),
```

while without it will be
```diff
-   next(50, "alpha")
+   next(50, "alpha"),
+   completed(50),
```

I'd say swiftlint is here to make it consistent across both codebase and tests. Again, this is a style preference so should be fine with any of them.

Reply:
Oh I see what you're saying here. 
I'm 100% fine with either. @fpillet do you have an opinion to here or there?

Reply:
Not really. Sometimes I'd leave trailing commas (i.e. if I'm in a rush) or while in the process of working on array items that I move around, removing the last trailing comma means more work to re-add it if I move it so ... well that's expert-level nitpicking.

The argument that I like is @mosamer 's reasoning about cleaner diffs. Looks like a win!

Reply:
Yup, Let me put those back in place :)

Reply:
@fpillet @mosamer should we add it in places where it didn't exist? there are some places there aren't any commas.

Even tho TBH these tests aren't supposed to change that much so not sure on the diff value 🤔 anyways, do it everywhere?

Reply:
Don't think so. Can't we just allow them?

Reply:
Oh yeah, we can definitely allow :) 
I just thought you want to retro-actively change. I'll just fix the rule. Thanks !